arm/Microsoft.Automation/automationAccounts/.parameters/encr.parameters.json

```diff
@@ -0,0 +1,207 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-wd-aut-encr-001"
+        },
+        // "schedules": {
```

Comment:
Do we need a schedule?

Reply:
Removed and tested  ( they were commented for testing purposes)